### PR TITLE
Bulk module author 2

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY

Following on from https://github.com/ansible/ansible/pull/48934

##### ISSUE TYPE
- Bugfix Pull Request

### CODE REVIEW

File: lib/ansible/modules/cloud/openstack/os_keystone_domain.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 module: os_keystone_domain
 short_description: Manage OpenStack Identity Domains
 author:
-    - Monty
+    - Monty (@emonty)
```
Comment (dagwieers) [inline lib/ansible/modules/cloud/openstack/os_keystone_domain.py]: Not the full name here ?

File: lib/ansible/modules/files/find.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 DOCUMENTATION = r'''
 ---
 module: find
-author: Brian Coca (based on Ruggero Marchei's Tidy)
+author: Brian Coca (@bcoca), based on Ruggero Marchei's Tidy
```
Comment (dagwieers) [inline lib/ansible/modules/files/find.py]: We normally don't add any additional meta information to the author's field.
I'd propose we add it as a comment, or as part of the copyright, or as a note ?

Comment (gundalow) [reply lib/ansible/modules/files/find.py]: Copyright already has

`# Copyright: (c) 2014, Ruggero Marchei <ruggero.marchei@daemonzone.net>`

So just listing bcoca as author

File: lib/ansible/modules/cloud/openstack/os_keystone_domain.py (hunk 1 above)
Comment (gundalow) [reply lib/ansible/modules/cloud/openstack/os_keystone_domain.py]: Good spot, thanks

**Comment by gundalow** [comment]
Helps reduce `label/needs_maintainer` https://github.com/ansible/community/issues/406

**Pull request merged**